packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6182,13 +6200,17 @@ export default class Meeting extends StatelessWebexPlugin {
    * @returns {undefined}
    */
   private handleShareAudioTrackEnded = async () => {
-    try {
-      await this.unpublishTracks([this.mediaProperties.shareAudioTrack]);
-    } catch (error) {
-      LoggerProxy.logger.log(
-        'Meeting:index#handleShareAudioTrackEnded --> Error stopping share: ',
-        error
-      );
+    if (this.wirelessShare && !this.mediaProperties.hasLocalShareTrack()) {
```

Comment:
this condition will always be false - at this point the audio or video share track is still set (same for handleShareVideoTrackEnded)

Reply:
oh you're right. I changed this to check for the other track - so in handleShareAudioTrackEnded I'm checking for shareVideoTrack to be undefined. In case we had both tracks, whichever handler gets called second will trigger the call to `leave()`. Otherwise, if we only have one track, that track's handler will call it as the other is undefined already